src/plugins/d3/renderer/hooks/useShapes/line/index.tsx

```diff
@@ -88,8 +108,7 @@ export const LineSeriesShapes = (args: Args) => {
             .attr('fill', 'none')
             .attr('stroke', (d) => d.color)
             .attr('stroke-width', (d) => d.width)
-            .attr('stroke-linejoin', 'round')
-            .attr('stroke-linecap', 'round');
+            .attr('stroke-dasharray', (d) => getLineDashArray(d.dashStyle, d.width));
```

Comment:
it is better to put `stroke-linejoin` and `stroke-linecap` in a separate setting and set the required default value for forms, if necessary